### ISSUE TITLE
Can't resolve 'path' 

### ISSUE THREAD

**Issue by mkrajcir** [postcss/postcss#1509]
I got an error with Angular 11 project
```
Error: ./node_modules/sanitize-html/node_modules/postcss/lib/input.js
Module not found: Error: Can't resolve 'path' in '/Users/mkrajcir/Devel/BMB/eshop/node_modules/sanitize-html/node_modules/postcss/lib'

Error: ./node_modules/sanitize-html/node_modules/postcss/lib/map-generator.js
Module not found: Error: Can't resolve 'path' in '/Users/mkrajcir/Devel/BMB/eshop/node_modules/sanitize-html/node_modules/postcss/lib'

Error: ./node_modules/sanitize-html/node_modules/postcss/lib/previous-map.js
Module not found: Error: Can't resolve 'path' in '/Users/mkrajcir/Devel/BMB/eshop/node_modules/sanitize-html/node_modules/postcss/lib'
```

**Comment by Semigradsky** [comment]
Also https://github.com/apostrophecms/sanitize-html/issues/431

Looks like `sanitize-html` used in the browser so node's dependencies not available.

**Comment by ai** [comment]
Talk to `sanitize-html` or Angular builder tool. I can’t help in invastigation.

Open an issue again if you will need some explicit changed in `postcss`.

**Comment by solblog** [comment]
I have exactly the same problem:
ERROR in ./node_modules/sanitize-html/node_modules/postcss/lib/input.js
Module not found: Error: Can't resolve 'path' in 'node_modules\sanitize-html\node_modules\postcss\lib'

**Comment by pento** [comment]
@ai: I think this is an issue in the same vein as #971, though it's likely that `path` would need a polyfill, rather than just being set to `false`. This is likely to show up more often, since [webpack 5 removed support for automatic polyfills](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed).

If you'd prefer not to add polyfills to `postcss`, it might be worth updating the [Browser docs](https://github.com/postcss/postcss/blob/main/README.md#browser) to note that webpack doesn't automatically polyfill node APIs any more. Perhaps noting the specific errors that folks might see would help it show up in Google searches. 🙂

For downstream folks wanting to fix this in their own repo, `browserify` _should_ work (though I haven't tested it). Alternatively, if you're using `webpack` installing `path-browserify` and adding this to your webpack config should fix it:

```js
resolve: {
	fallback: {
		path: require.resolve( 'path-browserify' ),
	},
},
```

**Comment by ai** [comment]
Polyfill for `path` will be too huge. We need to add `package.browser.path: null` (so even webpack 4 will not insert the polyfill) and prepare code to work without `path`.

I just do not have time since I work on another open-source project.

Please send PR (you can see how we deal with `fs` dependency as a reference).

**Comment by ai** [comment]
The fix by @mattcompiles was released in 8.2.7
